vyper/compile_lll.py

```diff
@@ -88,11 +91,13 @@ def apply_line_no_wrapper(*args, **kwargs):
 def compile_to_assembly(code, withargs=None, existing_labels=None, break_dest=None, height=0):
     if withargs is None:
         withargs = {}
-    assert isinstance(withargs, dict)
+    elif not isinstance(withargs, dict):
```

Comment:
kind of a nitpick but to preserve the semantics I think these should be standalone `if` statements, not `elif` statements. Somebody could come in later and change the body of the previous branch to read `withargs = []` and then we would no longer have the guarantee after this line that `withargs` is a dict.